spec/features/item_registration_csv_spec.rb

```diff
@@ -108,14 +98,28 @@
   end
 
   context 'when registration fails validation' do
-    it 'reports error' do
+    let(:non_default_apo) do
+      FactoryBot.create_for_repository(:persisted_apo,
+                                       label: 'My First APO',
+                                       roles: [
+                                         {
+                                           name: 'dor-apo-manager',
+                                           members: [{ identifier: 'sdr:administrator-role', type: 'workgroup' }]
+                                         }
+                                       ])
+    end
+
+    before do
+      non_default_apo # create a second APO to make sure the form does not reset to the default APO
+    end
+
+    it 'reports error, retains user values, and allows user to submit a corrected CSV' do
       visit registration_path
-      select '[Internal System Objects]', from: 'Admin Policy' # "uber APO"
+
+      select 'My First APO', from: 'Admin Policy'
       select 'registrationWF', from: 'Initial Workflow'
-      select 'book', from: 'Content Type'
-      select 'left-to-right', from: 'Viewing Direction'
-      select 'Stanford', from: 'View access'
-      select 'Stanford', from: 'Download access'
```

Comment:
is it useful to have a separate test with these values?  Not sure why we needed the change.  Simplifying?

Reply:
Choosing more non-default values to make sure they don't default when CSV validation fails.